ui/.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+bun.lockb
```

Comment:
I think the usual practice is to put lockfiles in git, please double check if this is needed